rtsp.go

```diff
@@ -510,22 +510,28 @@ func (rc *rtspCamera) initH265(session *description.Session) (err error) {
 			return
 		}
 
-		for _, nalu := range au {
-			frame, err := rc.rawDecoder.decode(nalu)
-			recoverableErr := &recoverableError{}
-			if errors.As(err, &recoverableErr) {
-				continue
-			}
-			if err != nil {
-				// This error is created with `github.com/pkg/errors`. Explicitly call `Error()` to
-				// avoid logging the stacktrace.
-				rc.logger.Debugw("error decoding(2) h265 rtsp stream", "err", err.Error())
-				return
+		// If the AU has more than one NALU, compact them into a single payload with NALUs seperated
+		// in AnnexB format. This is necessary because the H.265 decoder expects all NALUs for a frame
+		// to be in a single payload rather than chunked across multiple decode calls.
+		var packed []byte
```

Comment:
This makes a `[]byte(nil)` rather than a `[]byte{}`, is this what you  want? Both lengths are 0, so please check if there's anywhere we need to to a packed byte length check.